synchronizer/synchronizer.go

```diff
@@ -11,15 +10,12 @@ import (
 
 	"github.com/0xPolygonHermez/zkevm-node/etherman"
 	"github.com/0xPolygonHermez/zkevm-node/hex"
-	"github.com/0xPolygonHermez/zkevm-node/jsonrpc"
+	"github.com/0xPolygonHermez/zkevm-node/jsonrpc/types"
 	"github.com/0xPolygonHermez/zkevm-node/log"
-	"github.com/0xPolygonHermez/zkevm-node/sequencer/broadcast"
-	"github.com/0xPolygonHermez/zkevm-node/sequencer/broadcast/pb"
 	"github.com/0xPolygonHermez/zkevm-node/state"
 	"github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/core/types"
+	coreTypes "github.com/ethereum/go-ethereum/core/types"
```

Comment:
IMO core could refer to both zkEVM and Ethereum, I'd suggest:

```suggestion
	ethTypes "github.com/ethereum/go-ethereum/core/types"
```

and/or `zkTypes "github.com/0xPolygonHermez/zkevm-node/jsonrpc/types"`